nitro-protocol/test/contracts/ConsensusApp/latestSupportedState.test.ts

```diff
@@ -0,0 +1,113 @@
+import {expectRevert} from '@statechannels/devtools';
+import {Contract, Wallet, ethers, BigNumber} from 'ethers';
+
+import ConsensusAppArtifact from '../../../artifacts/contracts/ConsensusApp.sol/ConsensusApp.json';
+import {Channel, signState} from '../../../src';
+import {
+  getFixedPart,
+  getVariablePart,
+  SignedVariablePart,
+  State,
+} from '../../../src/contract/state';
+import {
+  deepEquals,
+  getTestProvider,
+  parseVariablePartEventResult,
+  setupContract,
+} from '../../test-helpers';
+const {HashZero} = ethers.constants;
+
+const provider = getTestProvider();
+let consensusApp: Contract;
+
+const participants = ['', '', ''];
+const wallets = new Array(3);
+const chainId = process.env.CHAIN_NETWORK_ID;
+const challengeDuration = 0x100;
+
+// Populate wallets and participants array
+for (let i = 0; i < 3; i++) {
+  wallets[i] = Wallet.createRandom();
+  participants[i] = wallets[i].address;
+}
+
+const channel: Channel = {chainId, channelNonce: 8, participants};
```

Comment:
Why does `channelNonce` is equal to 8?

Reply:
I just chose a random value, it shouldn't affect the test.